packages/@react-stately/slider/src/useSliderState.ts

```diff
@@ -244,6 +245,9 @@ export function useSliderState<T extends number | number[]>(props: SliderStateOp
     if (isDisabled || !isThumbEditable(index)) {
       return;
     }
+    if (dragging) {
```

Comment:
when dragging flipped to false, it was setting it with a stale value

Reply:
This is great! 

I was too fixated on updating `valuesRef.current` in `upateValue()` since all the thumb interactions must eventually arrive at `updateValue()`. Nice to know that there _is_ another place to put `valuesRef.current = values`!